README.md

```diff
@@ -31,22 +31,34 @@ After cloning the repo use it as you would use the package but import from your
 
 ```python
 from sqlalchemy import Column, Integer, String, ForeignKey
-from sqlalchemy.orm import relationship, declarative_base
-from rls.schemas import (
-    Permissive,
-    ExpressionTypes,
-    Command,
-)
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.sql import column
+from typing import Any
+from rls.schemas import Permissive, Command, ConditionArg
+
+# To avoid deletion by pre-commit hooks
```

Comment:
I don't understand this comment

Reply:
If I don't put the Any comment here , the precommit hooks delete the line causing it not to work correctly

Reply:
ah I see.

In any case, I think it is better practice to just do `import typing` and then use `typing.Any` directly.